project_4/index.html

```diff
@@ -0,0 +1,384 @@
+                    fair labor, and style. Zady was named of the world’s “Most Innovative
+                    Companies” in retail.</li>
+                <!-- image and information from https://truecostmovie.com/maxine-bedat-interview/ -->
+            </ul>
+
+            <ul class="speaker-info" data-aos="fade-up" data-aos-anchor-placement="top-center">
+                <a href="https://www.krisdedecker.org/" title="Link to Kris De Decker's Personal Website">
+                    <li><img src="images/kris.jpg" alt=""></li>
+                </a>
+                <li class="speaker-name">Kris De Decker</li>
+                <li class="speaker-title">Freelance Journalist & Creator of Low-Tech Magazine</li>
+                <li class="speaker-paragraph">Founder of the fully solar powered website running from a
+                    server on the roof of his apartment in Barcelona. The solar panel
+                    powers the battery to the server, and there is an icon on the site
+                    that shows users how much energy is left in the battery. This
+                    website has about 3 days a month where it actually shuts off
+                    because of the nature of solar power. This is a recent innovation,
+                    and has been widely talked about recently.
+                </li>
+                <li><a href=" https://solar.lowtechmagazine.com/"></a></li>
+                <!-- image and information from https://2019.theconference.se/speakers/kris-de-decker -->
+            </ul>
+
+
+
+            <ul class="speaker-info" data-aos="fade-up" data-aos-anchor-placement="top-center">
+                <a href="http://braungart.epea-hamburg.org/en/content/about-michael-braungart"
+                    title="Link to Michael Braungart's Personal Website">
+                    <li><img src="images/Michael_Braungart.jpg" alt=""></li>
+                </a>
+                <li class="speaker-name">Michael Braungart</li>
+                <li class="speaker-title">Chemist and Author</li>
+                <li class="speaker-paragraph">Author of the book Cradle to Cradle and The Upcycle: Beyond
+                    Sustainability--Designing for Abundance. Michael believes in
+                    redesigning industrial production to create a circular economy in
+                    which everything being made either gets reused into something
+                    else or goes back into the earth. They believe this is the way
+                    designers need to work in the future in order to keep humanity
+                    alive.</li>
+                <!-- image and information from https://epea-hamburg.com/michael-braungart/ -->
+
+            </ul>
+        </section>
+
+        <!-- SCHEDULE -->
+        <section id="schedule" class="schedule">
+            <h2 class="section-header">Schedule</h2>
+            <div class="left-friday">
+                <h4 data-aos="zoom-out-right">Friday, May 9th</h4>
```

Comment:
You have an `h2` and this is an `h4`, but no `h3` in this current structure (ie, under "Schedule").